ArduCopter/AP_Arming.cpp

```diff
@@ -445,7 +445,12 @@ bool AP_Arming_Copter::gps_checks(bool display_failure)
         AP_Notify::flags.pre_arm_gps_check = true;
         return true;
     }
-
+    // check for copter velocity innovations
```

Comment:
Nothing about this is Copter-specific.  QuadPlanes, for example, could also take advantage of this check.